src/main/resources/forge_at.cfg

```diff
@@ -157,6 +157,11 @@ public net.minecraft.client.renderer.WorldRenderer func_78909_a(I)I # getColorIn
 public net.minecraft.client.renderer.WorldRenderer func_178972_a(IIIII)V # putColorRGBA
 # ModelBlock Constructor
 public net.minecraft.client.renderer.block.model.ModelBlock <init>(Lnet/minecraft/util/ResourceLocation;Ljava/util/List;Ljava/util/Map;ZZLnet/minecraft/client/renderer/block/model/ItemCameraTransforms;)V
+# RenderManager
+public net.minecraft.client.renderer.entity.RenderManager field_178636_l # skinMap
+# RenderLivingEntity
+public net.minecraft.client.renderer.entity.RendererLivingEntity func_177094_a(Lnet/minecraft/client/renderer/entity/layers/LayerRenderer;)Z # addLayer
+public net.minecraft.client.renderer.entity.RendererLivingEntity func_177089_b(Lnet/minecraft/client/renderer/entity/layers/LayerRenderer;)Z # removeLayer
```

Comment:
Adding, i can get behind that. But removing is a no-go. Encouraging others to fuck with other's additions is stupid.
If you dont want your additional item to be rendered, just don't do it in your handler.